translations/de.ts

```diff
@@ -1037,7 +1041,7 @@ Disabling chat history now will leave the encrypted history intact (but not usab
         <location filename="../src/widget/form/settings/generalsettings.ui" line="119"/>
         <source>qTox will start minimized in tray.</source>
         <comment>toolTip for Start in tray setting</comment>
-        <translation>qTox minimiert starten.</translation>
+        <translation>qTox wird minimiert starten.</translation>
```

Comment:
Also laut comment im code ist das eben kein Knopf sondern schon der Tooltip. Du fragtest ja, ob es nen Tooltip gibt. Re Präsens und Futur. "Wenn ich diesen Knopf drücke passiert das" ist halt sehr lang. Ich persönlich bevorzuge Infinitive oder Imperative für Knöpfe oder Text, der direkt in den Einstellungen steht, z.B: "qTox minimiert starten", Präsenz für allgemeingültige Aussagen oder Dinge, die gerade geschehen: "qTox verbindet sich gerade mit der DHT ..." und Futur für zukünftige Konsequenzen von Einstellungen, die du vornimmnst (in den Tooltips): "Wenn diese Option aktiviert ist, wird qTox seine Farbe alle 0.3 SEkunden ändern."

Ich denke nicht, dass das Futur unverständlich ist. Jeder versthet doch "Morgen wird es regnen", und viele Apps haben Furut in den Tooltips und es ist auch im Original ein Futur.
Bzgl. Taskleiste: Ürsprünglich hatte ich ja nur "minimiert" drin. Wollte sie erst wieder aufgrund deines Vorschlags reinnehmen. Meine DE hat so eine, aber du hast Recht, dass, wenn es Umgebungen ohne solche gibt, es besser wäre, es so allgemein wie möglich zu halten. 

Daher für mein Dafürhalten ist für den Button "qTox minimiert starten"
und für den Tooltip "qTox wird minimirt starten"
oder alternativ das längere
"Wenn diese Option aktiviert ist / Wenn du diesen Knopf drückst, wird qTox (zunächst ohne geöffnetes Fenster, also) minimiert starten."